web/src/user_group_edit.js

```diff
@@ -362,7 +427,77 @@ function update_toggler_for_group_setting() {
     toggler.goto(select_tab);
 }
 
+function get_subgroup_names_for_status_text(subgroups) {
+    const subgroup_names = subgroups.map((subgroup) => subgroup.name);
+    if (Intl.ListFormat === undefined) {
+        return subgroup_names
+            .map((name) => `<b>${Handlebars.Utils.escapeExpression(name)}</b>`)
+            .join(", ");
+    }
+
+    const list_formatter = new Intl.ListFormat(user_settings.default_language, {
+        style: "long",
+        type: "unit",
+    });
+    const formatted_parts = list_formatter.formatToParts(subgroup_names);
+    return formatted_parts
+        .map((part) => {
+            if (part.type === "element") {
+                return `<b>${Handlebars.Utils.escapeExpression(part.value)}</b>`;
```

Comment:
We escape the name here as in template, we use `{{{` to not escape the bold tags.